src/hooks/useTranslator.ts

```diff
@@ -1,6 +1,6 @@
-import { useTranslation } from 'react-i18next'
+import { useTranslation as useTranslationImpl } from 'react-i18next'
```

Comment:
This renaming may not meet my original expectations. Because there are other places that use useTranslation directly from react-i18next, I call Translator here because I only want to use t in this function, and I don't want to write a deconstruction every time

这个重命名可能不符合我最初的期望。因为还有其他地方直接从react-i18next使用useTranslation，所以我在这里调用Translator，因为我只想在这个函数中使用t，并且我不想每次都写一个解构